From 4d885764f41be161c907cb8049d79cf0d6558981 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Wed, 30 Jan 2019 11:09:15 +0000
Subject: [PATCH] MANTA-4093 nginx should not modify its source tree during the
 build Reviewed by: Robert Mustacchi <rm@joyent.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 00000000..9544294d
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,2 @@
+/Makefile
+/objs
-- 
2.21.0

